third-party-crawler/third-party-crawler.py

```diff
@@ -0,0 +1,77 @@
+import csv
```

Comment:
Lets rename the file in line with source? in future if there are more third party crawlers, we can add more files and follow the naming based on the source of data?

Reply:
done here:
https://github.com/findthecluster/api-app/pull/1/commits/906ae201e013e5a237b2ae0c8185b9ddecc6eccd